src/views/Timeline.vue

```diff
@@ -42,42 +43,30 @@
               lazy-src="/img/lazy-load-bg.gif"
             >
               <template #placeholder>
-                    <v-row
-                    class="fill-height ma-0"
-                    align="center"
-                    justify="center"
-                    >
-                    <v-progress-circular
-                        indeterminate
-                        color="info lighten-2"
-                    />
-                    </v-row>
+                <v-row
+                  class="fill-height ma-0"
+                  align="center"
+                  justify="center"
+                >
+                  <v-progress-circular
+                    indeterminate
+                    color="info lighten-2"
+                  />
+                </v-row>
               </template>
               <v-row
                 class="fill-height ma-0"
                 align="start"
                 justify="start"
               >
-                <template
-                  v-if="isImageLoaded[index]"
-                >
+                <template v-if="isImageLoaded[index]">
                   <detection-boxes
                     :detections="sample.args.inference_result"
+                    :tensor_image_size="
```

Comment:
@vickywane can you please elaborate on this attribute? Why was it added and how is it used?